src/containers/TransferCurrency/DropdownForm.tsx

```diff
@@ -15,9 +20,16 @@ export const DropdownFormPresentation: React.SFC<Props> = props => {
     </option>
   ))
 
-  const isOnContactList = props.savedContacts.find(
-    ({ address }) => props.payeePublicKey === address,
-  )
+  const inactiveAccounts = getInactiveAccounts(props.accounts, props.activeAccount)
+  const inactiveAccountOptions = inactiveAccounts.map(({ name, address }) => (
```

Comment:
I think it would be nice to extract this option rendering into a function, maybe outside the component, which the `savedContacts` and `inactiveAccounts` can be mapped through. It's being called the same way twice (line 17 for `contactNames`) 
eg.
```typescript
  function renderOption({ address, name }) => (
    <option key={address} value={address}>
       { name }
    </option>
  )
```

Reply:
great point, thanks, I missed this one, changed it